msgid ""
msgstr ""
"Project-Id-Version: kolibri\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-11-01 22:12-0700\n"
"PO-Revision-Date: 2021-11-02 05:29\n"
"Last-Translator: \n"
"Language-Team: Urdu (Pakistan)\n"
"Language: ur_PK\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: kolibri\n"
"X-Crowdin-Project-ID: 201933\n"
"X-Crowdin-Language: ur-PK\n"
"X-Crowdin-File: /release-v0.15.x/django.po\n"
"X-Crowdin-File-ID: 4105\n"

#: core/auth/constants/commands_errors.py:17
msgid "Unexpected exception [{}]: {}"
msgstr "غیر متوقع اعتراض [{}]: {}"

#: core/auth/constants/commands_errors.py:20
msgctxt "Error report message when the allowed number of digits has been exceeded."
msgid "Content of cell '{}' is too long"
msgstr "سیل '{}' کا مواد بہت لمبا ہے"

#: core/auth/constants/commands_errors.py:24
msgctxt "Error report message when the cell contains an improper data format."
msgid "Content of cell '{}' is not valid"
msgstr "سیل '{}' کا مواد درست نہیں ہے"

#: core/auth/constants/commands_errors.py:26
msgid "Duplicated username"
msgstr "ڈپلیکیٹڈ صارف نام"

#: core/auth/constants/commands_errors.py:28
#: core/auth/management/commands/bulkimportusers.py:85
msgid "Username only can contain characters, numbers and underscores"
msgstr "صارف نام میں صرف حرف ، نمبر اور انڈر سکور ہوسکتے ہیں"

#: core/auth/constants/commands_errors.py:30
msgid "The column '{}' is required"
msgstr "کالم '{}' ضروری ہے"

#: core/auth/constants/commands_errors.py:31
msgid "Mix of valid and/or invalid header labels found in first row"
msgstr "درست اور / یا غلط ہیڈر لیبلوں کی آمیزہ پہلی قطار میں پائے گئے"

#: core/auth/constants/commands_errors.py:33
msgid "No default facility exists. Make sure to set up a facility on the device before running this command"
msgstr "کوئی پہلے سے طے شدہ سہولت موجود نہیں ہے۔ اس کمانڈ کو چلانے سے پہلے یقینی بنائیں کہ ڈیوائس پر کوئی سہولت قائم ہے"

#: core/auth/constants/commands_errors.py:35
#: core/auth/management/commands/bulkimportusers.py:99
msgid "Error trying to read csv file: {}"
msgstr "CSV فائل کو پڑھنے کی کوشش میں خامی: {}"

#: core/auth/constants/commands_errors.py:36
#: core/auth/management/commands/bulkimportusers.py:100
msgid "Error trying to write csv file: {}"
msgstr "CSV فائل لکھنے کی کوشش میں خامی: {}"

#: core/auth/management/commands/bulkexportusers.py:42
#: core/auth/management/commands/bulkexportusers.py:124
msgid "Database ID ({})"
msgstr "ڈیٹا بیس آئی ڈی ({})"

#: core/auth/management/commands/bulkexportusers.py:43
#: core/auth/management/commands/bulkexportusers.py:125
msgid "Username ({})"
msgstr "صارف نام ({})"

#: core/auth/management/commands/bulkexportusers.py:44
#: core/auth/management/commands/bulkexportusers.py:126
msgid "Password ({})"
msgstr "پاس ورڈ ({})"

#: core/auth/management/commands/bulkexportusers.py:45
#: core/auth/management/commands/bulkexportusers.py:127
msgid "Full name ({})"
msgstr "پورا نام ({})"

#: core/auth/management/commands/bulkexportusers.py:50
msgctxt "CSV column header for the type of user: ADMIN, LEARNER, COACH..."
msgid "User type ({})"
msgstr "صارف کی قسم ({})"

#: core/auth/management/commands/bulkexportusers.py:53
#: core/auth/management/commands/bulkexportusers.py:135
msgid "Identifier ({})"
msgstr "شناخت کنندہ ({})"

#: core/auth/management/commands/bulkexportusers.py:54
#: core/auth/management/commands/bulkexportusers.py:136
msgid "Birth year ({})"
msgstr "پیدائشی سال ({})"

#: core/auth/management/commands/bulkexportusers.py:55
#: core/auth/management/commands/bulkexportusers.py:137
msgid "Gender ({})"
msgstr "صنف ({})"

#: core/auth/management/commands/bulkexportusers.py:60
msgctxt "CSV column header for the list of classrooms names where the learner is going to be enrolled"
msgid "Learner enrollment ({})"
msgstr "متعلم کا اندراج ({})"

#: core/auth/management/commands/bulkexportusers.py:67
msgctxt "CSV column header for the list of classrooms names where the tutor is going to be a coach"
msgid "Coach assignment ({})"
msgstr "کوچ اسائنمنٹ({})"

#: core/auth/management/commands/bulkexportusers.py:132
msgctxt "CSV header for the type of user: ADMIN, LEARNER, COACH..."
msgid "User type ({})"
msgstr "صارف کی قسم ({})"

#: core/auth/management/commands/bulkexportusers.py:142
msgctxt "CSV file header for the list of classrooms names where the learner is going to be enrolled"
msgid "Learner enrollment ({})"
msgstr "متعلم کا اندراج ({})"

#: core/auth/management/commands/bulkexportusers.py:149
msgctxt "CSV file header for the list of classrooms names where the tutor is going to be a coach"
msgid "Coach assignment ({})"
msgstr "کوچ اسائنمنٹ({})"

#: core/auth/management/commands/bulkimportusers.py:76
msgctxt "Error message that might appear when there's a programming error importing a CSV file"
msgid "Unexpected error [{}]: {}"
msgstr "غیر متوقع خرابی [{}]: {}"

#: core/auth/management/commands/bulkimportusers.py:80
msgctxt "Error when the command is executed in the Terminal (command prompt)"
msgid "Value in column '{}' has too many characters"
msgstr "کالم '{}' کی قدرمیں بہت زیادہ حرف ہیں"

#: core/auth/management/commands/bulkimportusers.py:82
msgid "Value in column '{}' not valid"
msgstr "کالم '{}' میں ویلیو درست نہیں ہے"

#: core/auth/management/commands/bulkimportusers.py:83
msgid "Username is duplicated"
msgstr "صارف نام ڈپلیکیٹ ہے"

#: core/auth/management/commands/bulkimportusers.py:89
msgctxt "Error message indicating that the CSV file selected for import is missing a required column"
msgid "The column '{}' is required"
msgstr "کالم '{}' ضروری ہے"

#: core/auth/management/commands/bulkimportusers.py:93
msgctxt "Error message indicating that one column header in the CSV file selected for import is missing or incorrect"
msgid "Incorrect header label found in the first row"
msgstr "پہلی صف میں غلط ہیڈر لیبل ملا ہے"

#: core/auth/management/commands/bulkimportusers.py:97
msgctxt "Error when the command is executed in the Terminal (command prompt)"
msgid "No default facility exists. Make sure to set up a facility on the  device before importing users and classes"
msgstr "کوئی پہلے سے طے شدہ سہولت موجود نہیں ہے۔ صارفین اور کلاسوں کو درآمد کرنے سے پہلے ڈیوائس پر ایک سہولت قائم کرنا یقینی بنائیں"

#: core/auth/management/commands/bulkimportusers.py:102
msgid "The password field is required. To leave the password unchanged in existing users, insert an asterisk (*)"
msgstr "پاس ورڈ فیلڈ درکار ہے۔ موجودہ صارفین میں پاس ورڈ کو تبدیل نا کرنے کے لئے ، نجمہ داخل کریں (*)"

#: core/auth/management/commands/bulkimportusers.py:105
msgid "Cannot update user with ID: '{}' because no user with that database ID exists in this facility"
msgstr "ID کے ساتھ صارف کو اپ ڈیٹ نہیں کیا جاسکتا: '{}' کیونکہ اس سہولت میں ڈیٹا بیس آئی ڈی والا کوئی صارف موجود نہیں ہے"

#: core/auth/management/commands/bulkimportusers.py:107
msgid "Database ID is not valid"
msgstr "ڈیٹا بیس آئی ڈی درست نہیں ہے"

#: core/content/api.py:170
msgid "Resource"
msgstr "وسائل"

#: core/content/api.py:1327
msgid "The requested channel does not exist on the content server"
msgstr "مواد کے سرور میں التجا ہوا چینل موجود نہیں ہیں"

#: core/content/models.py:252
msgid "Unknown format"
msgstr "نا معلوم فورمیٹ"

#: core/device/serializers.py:36
msgid "Language is not supported by Kolibri"
msgstr "کولیبری کی طرف سے زبان کی معاونت نہیں"

#: core/logger/csv_export.py:205
msgctxt "Default name for the exported CSV file with content session logs. Please keep the underscores between words in the translation"
msgid "content_session_logs"
msgstr "مواد_اجلاس_لاگز"

#: core/logger/csv_export.py:215
msgctxt "Default name for the exported CSV file with content summary logs. Please keep the underscores between words in the translation"
msgid "content_summary_logs"
msgstr "مواد_سمری_لاگز"

#: core/tasks/api.py:82
msgid "There was a network error."
msgstr "نیٹ ورک میں ایک نقص تھا."

#: core/tasks/api.py:84
msgid "There was a disk access error."
msgstr "ڈسک کی رسائی میں ایک نقص تھا."

#: core/tasks/api.py:86
msgid "There was an unknown error."
msgstr "ایک نامعلوم نقص تھا."

#: core/templates/kolibri/base.html:13
#: core/templates/kolibri/unsupported_browser.html:13
msgid "Kolibri"
msgstr "کولیبری"

#: core/templates/kolibri/unsupported_browser.html:42
msgid "Unsupported browser"
msgstr "غیر تائید شدہ براؤزر"

#: core/templates/kolibri/unsupported_browser.html:43
msgid "Sorry, your browser version is not supported."
msgstr "معاف کیجئے گا, آپ کے براؤزر ورژن معاون نہیں."

#: core/templates/kolibri/unsupported_browser.html:44
msgid "To use Kolibri, we recommend using Firefox or Chrome."
msgstr "کولیبری استعمیل کرنے کے لیے ہم فائر فکس یا کروم استعمال کرنے کی مشورہ دیتے ہے."

#: core/templates/kolibri/unsupported_browser.html:45
msgid "You can also try updating your current browser."
msgstr "آپ اپنے موجودہ براؤزر کو اپ ڈیٹ کرنے کی بھی کوشش کر سکتے ہیں."

#: core/views.py:168
msgid "No appropriate redirect pages found. It is likely that Kolibri is badly configured"
msgstr "کوئی مناسب صفحات نہیں ملے۔ ممکن ہے کہ یہ کولیبری بری طرح تشکیل کیا گیا ہے"

#: plugins/facility/views.py:52
msgctxt "Default name for the exported CSV file of facility user data. Please keep the underscore between words in the translation"
msgid "users_{}"
msgstr "صارفین_ {}"

